src/main/java/com/dreamhorizon/enchantments/enchantments/Cannibalism.java

```diff
@@ -64,7 +64,8 @@ public static void addCannibalismEffect(ItemStack attackItem, LivingEntity damag
         } else {
             damager.setHealth(damager.getHealth() + healthToGain);
         }
-        
+
+        //TODO: Fix - Doesn't seem to work
         damager.addPotionEffect(new PotionEffect(PotionEffectType.SATURATION, saturationDuration, 0)); //Add food to user
```

Comment:
change the 0 at the end to a 1, you are giving them saturation 0 which is nothing.

Reply:
the same is true for battlerush enchantment